exchanges/graphcache/src/types.ts

```diff
@@ -692,6 +707,11 @@ export type ResolverConfig = {
   } | void;
 };
 
+// TODO: docs
```

Comment:
The TSDocs for this are going to be highly similar to resolvers but I think that's all right as this might replace resolvers over time